Mono.Cecil.Cil/Symbols.cs

```diff
@@ -802,6 +809,45 @@ public ISymbolReader GetSymbolReader (ModuleDefinition module, string fileName)
 			return null;
 		}
 
+		public static SymbolReaderKind GetSymbolReaderKind(string fileName)
+		{
+			try
+			{
```

Comment:
ReadAssembly can throw for various reasons (no file, corrupt file, not managed, etc).  I think having a "None" enum is simpler and easier to deal with than the Try method pattern for this case.  What value would the Try pattern have?  The caller would then need additional logic to detect the error and come to the same conclusion: no symbols.